pep_sphinx_extensions/pep_theme/static/style.css

```diff
@@ -201,13 +184,6 @@ a img {
 }
 
 /* List rules */
```

Comment:
I'd like to retain this, I think the square bullets look better, and are more consistent than the browser default for two levels of indentation:

```suggestion
/* List rules */
ul {list-style: square}
```


Current on left, this PR on right:

![image](https://github.com/python/peps/assets/9087854/4eca5451-cbb7-45f4-9657-c37d17bf59a6)

Reply:
Is the new main section narrower or just looks that way?  With the extra spacing it keeps less content per screen.

Reply:
Interesting, I find having the same style bullets for all indents make it _harder_ to read and figure about.

The different styles make it easier to see what's going on, especially for long lists with at least four levels of indentation, such as:

* https://peps.python.org/pep-0703/
* https://pep-previews--3132.org.readthedocs.build/pep-0703/

Reply:
> Is the new main section narrower or just looks that way? With the extra spacing it keeps less content per screen.

They're the same. Maybe the side-by-side windows in the screenshot were slightly different sizes. Adding background colour for full-screen windows:

<details>
<summary>Details</summary>

<img width="1582" alt="image" src="https://github.com/python/peps/assets/1324225/1d33d330-722a-4a79-aac5-cdd18749b23c">

<img width="1582" alt="image" src="https://github.com/python/peps/assets/1324225/e22a05b0-82ac-4937-9a4f-30e0575c3f6c">


</details>

Reply:
> The different styles make it easier to see what's going on, especially for long lists with at least four levels of indentation

I've taken another side-by-side (below, in the details block).

From my perspective, the apparent font size of the sidebar is smaller on the right (this PR) than on the left (current style). Perhaps we should increase the font size in this PR such that the (apparent) font sizes are similar?

Also, on FireFox, the default styling only appears to define three levels of unique bullets, so at the fourth level ('Default ``(0b00)``') the bullets are the same as the third. I (personally) find the homogeneity of the current style easier to use, though perhaps readability improvements could be to slightly increse vertical whitespace?

<details>
<summary>Side-by-side</summary>

![image](https://github.com/python/peps/assets/9087854/0b3b8070-cd20-4810-a8ea-a2ea75b8cdf1)

</details>

A

Reply:
I agree with Hugo's take that different bullet point styles per level are more readable.